docs/serving/installing-logging-metrics-traces.md

```diff
@@ -17,6 +17,15 @@ sections to do so now.
 
 ## Metrics
 
+1. Run the following command to enable request metrics if they are wanted:
```

Comment:
If you run this command, aren't you opening an editor to update the configs manually? IE, running this command doesn't automatically enable anything; you have to follow the instructions below to enable metrics? If that's the case, I think we should say that.

Reply:
Done.